v03_pipeline/bin/dataproc_vep_init.bash

```diff
@@ -16,6 +16,7 @@ set -x
 export PROJECT="$(gcloud config get-value project)"
 export ENVIRONMENT="$(/usr/share/google/get_metadata_value attributes/ENVIRONMENT)"
 export REFERENCE_GENOME="$(/usr/share/google/get_metadata_value attributes/REFERENCE_GENOME)"
+export APP_VERSION="$(/usr/share/google/get_metadata_value attributes/APP_VERSION)"
```

Comment:
Pattern here will be to extract the `APP_VERSION` from within the docker container, pass that into the dataproc init, and then reference it here 😬 .